docs/src/get-started/tutorials/understanding-hypers.py

```diff
@@ -171,12 +172,12 @@
 # .. math::
 #
 #     f(r) = \begin{cases}
-#           1 &r < r_c - s,\\
-#           0.5 + 0.5 \cos[\pi (r - r_c + s) / s] &r_c - s < r <= r_c, \\
+#           1 &r < r_c - w,\\
```

Comment:
If we change it here then also in he doc string of the rust function.

Reply:
So I undid the changes in the rust docstring, since these are also used here: https://luthaf.fr/rascaline/latest/references/calculators/spherical-expansion.html, and were not rendering properly.